ops/testing.py

```diff
@@ -1882,6 +1882,24 @@ class _Secret:
     grants: Dict[int, Set[str]] = dataclasses.field(default_factory=dict)
 
 
+class RebootingMachineError(Exception):
```

Comment:
Thanks for thinking about the testing / Harness side of things!

I think the approach of raising an exception is good. I'm not sure about the name, as it's more of a signal than an "error" like `FileNotFoundError`. What do you think of `MachineRebooting`, or even `RebootNow`?

Reply:
> I think the approach of raising an exception is good. I'm not sure about the name, as it's more of a signal than an "error" like `FileNotFoundError`. What do you think of `MachineRebooting`, or even `RebootNow`?

Yeah, it was without the error until the linter complained, so I renamed it rather than silence the warning. I'm happy to go back. `RebootNow` works for me.